nosqlbench/http-jsonapi-crud-dataset.yaml

```diff
@@ -1,17 +1,20 @@
 min_version: "5.17.3"
 
+# Example command line (when Stargate is running on localhost):
+# nb5 -v http-jsonapi-crud-dataset jsonapi_host=localhost docscount=20000 threads=20 dataset=dataset.txt
+
 description: >2
   This workload emulates CRUD operations for the Stargate Documents API.
-  It requires a data set file, where each line is a single JSON document to be used for writes and updates.
+  It requires a data set file (default dataset.txt), where each line is a single JSON document to be used for writes and updates.
   Note that jsonapi_port should reflect the port where the Docs API is exposed (defaults to 8181).
 
 scenarios:
   default:
     schema:   run driver=http tags==block:schema threads==1 cycles==UNDEF
     write:    run driver=http tags==name:"write.*" cycles===TEMPLATE(write-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
```

Comment:
is 10M docs as default meaningful?

Reply:
Will make is to 1M, in case file has less records it tries to reload the same data with different id.